website/docs/r/service_connector_app_service.html.markdown

```diff
@@ -109,6 +109,8 @@ The following arguments are supported:
 
 * `vnet_solution` - (Optional) The type of the VNet solution. Possible values are `serviceEndpoint`, `privateLink`.
 
+* `key_vault_id` - (Optional) The key vault id to store secret.
```

Comment:
should this be
```suggestion
* `secret_store_key_vault_id` - (Optional) The key vault id to store secret.
```

Reply:
is this to store the secret store in? load the secret store from? could we expand the description here some

Reply:
Will change.